### PR TITLE
移除无效的条目 (fixes #124)

### LINKED ISSUE #124: 无效词条 “大波Man”
### 問題描述

```
E20210703 03:45:29.408849   961 entry_collector.cc:135] Encode failure: '大波Man'.
E20210703 03:45:43.208457   962 entry_collector.cc:135] Encode failure: '大波Man'.
```

此词条在昨日的修改 https://github.com/rime/rime-cantonese/commit/346b7524e708b1cb7e2a673f84cd06479667b902 中引入。

### PR DESCRIPTION
**Pull request by felixonmars**

### CODE REVIEW

File: jyut6ping3.phrase.dict.yaml [hunk 1]
```diff
@@ -347949,7 +347949,6 @@ sort: by_weight
 基進黨
 大中華膠
 大外宣
-大波Man
```
Comment (ayaka14732) [inline jyut6ping3.phrase.dict.yaml:347952]: Add pronunciation daai6 bo1 men1 and move to `jyut6ping3.lettered.dict.yaml`.

Comment (ayaka14732) [inline jyut6ping3.phrase.dict.yaml:347952]: Add pronunciation daai6 bo1 men1 and move to `jyut6ping3.lettered.dict.yaml`.

**Review by ayaka14732 — CHANGES_REQUESTED** [review]

**Comment by ayaka14732** [comment]
@laubonghaudoi why merged

**Comment by laubonghaudoi** [comment]
@ayaka14732 嗰個詞條含有英文單詞，應該放入 .lettered

**Comment by tanxpyox** [comment]
@ayaka14732 Do you think we can set up a CI flow to check for these errors with https://github.com/rime/librime/blob/master/tools/rime_deployer.cc#L89 ?

e.g. 

```sh
sudo apt install ibus-rime # can probably get away with only librime1, but just to be sure
set -e 
bash rime-install cantonese # add the other prerequisites if needed
rime-deployer --build ${rime_dir} # replace ${rime_dir} with plum target folder
```

**Pull request merged**